Packet++/header/TcpLayer.h

```diff
@@ -464,6 +602,14 @@ namespace pcpp
 		 */
 		bool removeTcpOption(TcpOptionType optionType);
```

Comment:
Shouldn't we mark this method as deprecated also?

Reply:
Good catch!